packages/side-nav/src/SideNav.story.tsx

```diff
@@ -17,7 +17,7 @@ function MockSideNav() {
     <SideNav>
       <SideNavGroup header={textHeader}>
         <SideNavItem active>Active State</SideNavItem>
-        <SideNavItem disabled>Disabled State</SideNavItem>
+        <SideNavItem>Disabled State</SideNavItem>
```

Comment:
I believe you are missing the `disabled` prop in this demo.

Reply:
you're right, thank you! added back in